docs/equiv-check/index.md

```diff
@@ -0,0 +1,133 @@
+
+```{note}
+We are working on extending this to support functions with side effects;
+  for the remaining documentation,
+  we assume a user is interested in running the equivalence checker
+  for two pure functions which may belong to the same contract
+  or two different contracts.
+```
+
+The equivalence checker front-end automatically generates (1) a
+  CVL spec to check if two functions are equivalent, and, (2) a
+  configuration file (`.conf`) for running the Certora Prover.
+
+## Installation
+
+The front-end script, `certoraEqCheck`,
+  should be available as part installing `certora-cli`.
+
+
+(example)=
+## Example
+
+Consider two contracts, `BasicMathGood.sol` and `BasicMathBad.sol` shown
+  below with two functions, `add` and `add_mult`.
+
+```
+BasicMathGood.sol:
+------------------
+
+contract BasicMathGood {
+    function add(uint256 a, uint256 b) public pure returns(uint256) {
+        return a + b;
+    }
+}
+
+```
+
+```
+BasicMathBad.sol:
+------------------
+
+contract BasicMathBad {
+    function add_mult(uint256 a, uint256 b) public pure returns(uint256) {
+        return a * b;
+    }
+}
+```
+
+We are interested in checking the equivalence of `add` and `add_mult`.
+In this simple case, these two functions are clearly not equivalent
+  but you can imagine scenarios where
+  the functions are more complex.
+Equivalence checking can be used to check whether two functions that
+  may be implemented differently, are still semantically equivalent.
+
+## Usage
+
+`certoraEqCheck` can be run either in default (`def`) mode,
+ in which the user must supply all the required information as
+ command line arguments (see below),
+ or in a `conf` mode where the user supplies a
+ Certora Prover `conf` file along with additional arguments.
+
+### Default mode
+
+To run the equivalence checker in default mode,
+  use `certoraEqCheck` like so:
+
+```bash
+certoraEqCheck def "path_to_file:contract:function:solc" "path_to_file:contract:function:solc"
+```
+
+For the functions in {ref}`example`, this would look like so:
+
+```bash
+certoraEqCheck def Test/EqCheck/BasicMathGood.sol:add:solc8.0 Test/EqCheck/BasicMathBad.sol:add_pass:solc8.0
+```
+
+In the above example, `solc` is the name of the executable
+  for the Solidity compiler version you are using.
+The Solidity compilers do not need to be the same for both arguments to
+ `certoraEqCheck`, it only need to be appropriate for the given contract.
+Also note how
+  the contract field can be omitted if the contract name is the same
+  the file name.
+
+
+### Configuration mode
+
+To run the equivalence checker in the configuration mode,
+  use `certoraEqCheck` like so:
+
+```bash
+certoraEqCheck conf <path_to_conf>.conf contract:function contract:function
+```
+
+For the functions in {ref}`example`, this would look like so:
+
+```bash
+   certoraEqCheck conf Test/EqCheck/testGood.conf BasicMathGood:add BasicMathBad:add_mult
+```
+
+where `testGood.conf` contains the following configuration:
```

Comment:
Is this a standard `certoraRun` config file?  If so, it's probably worth saying that.  I would normally link to the documentation, except we don't have it right now. It might be worth creating a placeholder page with a {todo} and linking to that.  Maybe ask @rahav-certora - I think he maybe is working on one?

Reply:
ok i asked him and added a clarification for now